tests/integration/test_modelrun_components.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+import predictsignauxfaibles.__main__ as main
+from predictsignauxfaibles.data import SFDataset, OversampledSFDataset
+
+
+class ParserTest(unittest.TestCase): # pylint: disable=too-few-public-methods
+    """
+    Class to test functionnalities of the parser we build
+    to process optionnal arguments
+    """
+
+    def setUp(self):
+        self.parser = main.make_parser()
```

Comment:
I don't understand what this is doing
(and our testing framework is `pytest`, not `unittest`)

Reply:
This will only be useful for issue #49
Removing it for now, I will add it back and add test methods to `ParserTest` on another branch (using `pytest`)